frontend-react/src/pages/admin/value-set-editor/ValueSetsDetail.tsx

```diff
@@ -130,7 +130,7 @@ const ValueSetsDetailHeader = ({
     );
 };
 
-const ValueSetsDetail = () => {
+export const ValueSetsDetail = () => {
```

Comment:
This is from merging main into this branch, I believe. This may have merged a newer master than `5557-main`

Reply:
Ah, no, sorry, this was because of a weird bug where typescript detected two default exports...